recipes/flashlfq/elements.dat

```diff
@@ -0,0 +1,2878 @@
+<html>
+<head>
```

Comment:
Why is this file needed here @jj-umn? Can we store it on cargo-port?

Reply:
@bgruening The FlashLFQ requires that file and expects it in the same dir as the executable.  I asked the code developer to include it in the distribution for the next release.